cmd/tailscale/cli/file.go

```diff
@@ -286,22 +287,119 @@ func runCpTargets(ctx context.Context, args []string) error {
 	return nil
 }
 
+type existingFileBehavior int
+
+const (
+	FailOnExist         existingFileBehavior = iota
```

Comment:
Exported consts of an unexported type is a bit weird in Go.

But also this is just a bit overkill. The strings are fine. Let's ditch the type here and the map on line 298 and the String method, etc.